src/pulselistener/pulselistener/lib/bus.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import asyncio
+import inspect
+import multiprocessing
+
+from cli_common.log import get_logger
+
+logger = get_logger(__name__)
+
+
+class MessageBus(object):
+    '''
+    Communication bus between processes
+    '''
+    def __init__(self, max_messages=None):
+        self.queues = {}
+        self.max_messages = max_messages
+        self.nb_messages = 0
+
+    def add_queue(self, name, mp=False):
+        '''
+        Create a new queue on the message bus
+        * asyncio by default
+        * multiprocessing when mp=True
+        '''
+        assert name not in self.queues, 'Queue {} already setup'.format(name)
+        if mp:
+            self.queues[name] = multiprocessing.Queue()
+        else:
+            self.queues[name] = asyncio.Queue()
+
+    def is_alive(self):
```

Comment:
This function is checking whehter the queue is not full basically, maybe call it something like `is_not_full` or `is_full` (and make the callers check not is_full).